lib/taurus/qt/qtgui/display/tauruslabel.py

```diff
@@ -448,9 +456,15 @@ def resetAutoTrim(self):
         self.setAutoTrim(self.DefaultAutoTrim)
 
     def displayValue(self, v):
+        """Reimplementation of displayValue for TaurusLabel"""
         if self._permanentText is not None:
-            return self._permanentText
-        return TaurusBaseWidget.displayValue(self, v)
+            value = self._permanentText
+        else:
+            value = TaurusBaseWidget.displayValue(self, v)
+
+        attr = self.getModelObj()
+        dev = attr.getParent()
+        return value.format(dev=dev, attr=attr)
```

Comment:
protect this against exceptions in format(). Something like:
```python
try:
    v = value.format((dev=dev, attr=attr))
except Exception as e:
    self.warning("Error formatting display (%r). Reverting to raw string", e)
    v = value
return v
```

Reply:
Ok